cmd/janitor/gcp_janitor.py

```diff
@@ -558,7 +563,7 @@ def main(project, days, hours, filt, rate_limit, service_account, additional_zon
     for api, resources in RESOURCES_BY_API.items():
         if not api_enabled(project, api):
             continue
-        if api == 'staging-gkehub.sandbox.googleapis.com' or api == 'gkehub.googleapis.com':
+        if api == 'staging-gkehub.sandbox.googleapis.com' or api == 'gkehub.googleapis.com' or api == 'autopush-gkehub.sandbox.googleapis.com':
```

Comment:
this is basically fine, can it be simplified somehow?

Reply:
Added a set for gkehub apis, not an expert in python, please check :).